tt_eager/tt_lib/csrc/operations/primary/transformers/module.hpp

```diff
@@ -120,6 +120,7 @@ void py_module(py::module& m_transformers) {
         py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
         py::arg("program_config").noconvert() = SDPADefaultProgramConfig{},
         py::arg("compute_kernel_config").noconvert() = std::nullopt,
+        py::arg("valid_seq_len").noconvert() = std::nullopt,
```

Comment:
What does `valid_seq_len` do?

Reply:
This op takes in the `k_cache` and `v_cache` tensors, so `valid_seq_len` tells the op how much of the K and V cache to read to compute the output.

We're going to iterate on this as we add support for users on different token indexes and paged KV cache, but this is how we'll use this first version of the op.